.github/workflows/ci.yml

```diff
@@ -156,6 +156,14 @@ jobs:
   test_features:
     name: Feature Tests
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        dirname: [agents, "*"]
+        include: 
```

Comment:
include permet de fournir des paramètres supplémentaires à la combinaise de matrice.
Si on met en direct `command : [ a, b]` ici sans le nester sous `include`, alors on va lancer 4 combinaisons : `agent-a, agent-b, *-a, *-b`.
En utilisant `include`, on ne lance que deux combinaisons : ` agent (a), * (b)`

cf https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/running-variations-of-jobs-in-a-workflow#expanding-or-adding-matrix-configurations